docs/sources/alerting/configure-notifications/manage-contact-points/_index.md

```diff
@@ -46,28 +56,29 @@ refs:
       destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/manage-contact-points/integrations/configure-slack/
     - pattern: /docs/grafana-cloud/
       destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/manage-contact-points/integrations/configure-slack/
+  external-alertmanager:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/set-up/configure-alertmanager/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/set-up/configure-alertmanager/
 ---
 
 # Configure contact points
 
 Use contact points to select your preferred communication channel for receiving notifications when your alert rules are firing. You can add, edit, delete, export, and test a contact point.
 
-{{% admonition type="note" %}}
-You cannot delete contact points that are in use by a notification policy. Either delete the notification policy or update it to use another contact point.
-{{% /admonition %}}
-
 On the **Contact Points** tab, you can:
 
-- Search for name and type of contact points and integrations
-- View all existing contact points and integrations
-- View how many notification policies each contact point is being used for and navigate directly to the linked notification policies
-- View the status of notification deliveries
-- Export individual contact points or all contact points in JSON, YAML, or Terraform format
-- Delete contact points that are not in use by a notification policy
+- Search for name and type of contact points and integrations.
+- View all existing contact points and integrations.
+- View how many notification policies each contact point is being used for and navigate directly to the linked notification policies.
+- View the status of notification deliveries.
+- Export individual contact points or all contact points in JSON, YAML, or Terraform format.
+- Delete contact points. Note that you cannot delete contact points that are in use by a notification policy. To proceed, either delete the notification policy or update it to use another contact point.
```

Comment:
The admonition was very prominent at the top, so I decided to replace the `admonition` note with an inline message.

Alternatively, we could create a section `Delete a contact point` with instructions and include the note there.